src/panels/lovelace/special-rows/hui-divider-row.ts

```diff
@@ -0,0 +1,55 @@
+import { html, LitElement } from "@polymer/lit-element";
+import { EntityRow, DividerConfig } from "../entity-rows/types";
+import { HomeAssistant } from "../../../types";
+import { TemplateResult } from "lit-html";
+
+class HuiDividerRow extends LitElement implements EntityRow {
+  public hass?: HomeAssistant;
+  private _config?: DividerConfig;
+
+  static get properties() {
+    return {
+      _config: {},
+    };
+  }
+
+  public setConfig(config): void {
+    if (!config) {
+      throw new Error("Error in card configuration.");
+    }
+
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config) {
+      return html``;
+    }
+
+    return html`
+      ${this._createDivider()}
```

Comment:
Why extract into a new method?

Reply:
I couldn't figure out how to apply the passed into styles to the div without using `setProperty` 🤷‍♂️